content/css/jenkins.css

```diff
@@ -1,18 +1,52 @@
 #ji-hover-layer {position:relative; z-index:999}
-body > .popover {width:350px;min-width:350px !important; max-width:90% !important}
+body .popover.download-options {width:620px;min-width:500px !important; max-width:90% !important; box-shadow:0 3px 15px rgba(0,0,0,.25)}
 #ji-toolbar {position:fixed; top:0; left:0; right:0; z-index:100; border-radius:0}
+#ji-toolbar > .navbar-brand {font:bold 2rem 'georgia'; margin-top:-.2rem;}
+#ji-toolbar > .navbar-brand,
+#download-menu{ margin:0 0 0 -99rem; overflow:hidden;}
+body h1.page-title {font:bold 3rem 'georgia'; margin-top:-.2rem}
+.sub-heading {margin:-.5rem 0 1rem;}
+#ji-toolbar.navbar-dark .navbar-nav .nav-link {color:rgba(255,255,255,.85)}
+#ji-toolbar.navbar-dark .navbar-nav .nav-link:hover {color:rgba(255,255,255,.95)}
+#ji-toolbar form {white-space:nowrap}
+#ji-toolbar.bg-inverse {background:#000}
 #ji-toolbar-offset {min-height:3.4rem}
 #ji-top-peg {position:absolute; right:0;text-align:center; left:0; z-index:99}
 #ji-top-peg #ji-fork-tag {position:fixed; right:0}
-.carousel-item {min-height:450px; background:#ccc; box-shadow:inset 0 0 200px 10px rgba(0,0,0,.1), inset 0 150px 200px -100px rgba(0,0,0,.33), inset 0 -150px 200px -100px rgba(0,0,0,.33)}
+.carousel-item {min-height:450px; background:#fff; box-shadow:inset 0 0 200px 10px rgba(0,0,0,.1), inset 0 150px 200px -100px rgba(0,0,0,.33), inset 0 -150px 200px -100px rgba(0,0,0,.33)}
 .carousel-item > .carousel-caption {bottom:auto; top:1.5rem}
-#ji-download {top:275px; right:0; left:0; position:absolute; z-index:9; text-align:center }
+#ji-download {top:300px; right:0; left:0; position:absolute; z-index:9; text-align:center }
 #download-menu .dropdown-toggle,
 #ji-download .dropdown-toggle {box-shadow:inset 0 0 0 200rem rgba(0,0,0,.2)}
 #ji-download .carousel-caption{ position:static; display:block; padding:1rem;}
 
+.carousel-item video {position:absolute; top:0; left:0; right:0; bottom:0; width:100%; opacity:.67
```

Comment:
In the future, if CSS is being added, I would like to see some code-type comments to make sure that future contributors don't inadvertently break CSS when updating